ksql-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -123,7 +123,17 @@
     },
     {
       "name": "validate int elements OK",
-      "format": ["DELIMITED", "KAFKA"],
+      "format": ["DELIMITED"],
+      "statements": [
+        "CREATE STREAM INPUT (V0 INT) WITH (kafka_topic='input', value_format='{FORMAT}');",
+        "CREATE STREAM OUTPUT AS SELECT * FROM INPUT;"
+      ],
+      "inputs": [{"topic": "input", "value": "10"}],
+      "outputs": [{"topic": "OUTPUT", "value": "10"}]
+    },
+    {
+      "name": "validate int elements OK",
+      "format": ["KAFKA"],
```

Comment:
I split out KAFKA and DELIMITED formats because they serialize slightly differently in tests